### DISCUSSION TITLE
字号

### DISCUSSION THREAD

**Discussion by netnings** [tamlok/vnote#1815] [General]
请问编辑字号和显示字号，代码字号及字体怎么修改呢

**Comment by tamlok** [comment]
https://tamlok.gitee.io/vnote/zh_cn/#!docs/%E7%94%A8%E6%88%B7/%E4%B8%BB%E9%A2%98%E5%92%8C%E6%A0%B7%E5%BC%8F.md

**Reply by netnings** [reply]
里面的代码没有注释，具体修改哪几个字体呢

**Reply by tamlok** [reply]
添加了示例： https://vnotex.github.io/vnote/zh_cn/#!docs/%E7%94%A8%E6%88%B7/%E4%B8%BB%E9%A2%98%E5%92%8C%E6%A0%B7%E5%BC%8F.md#toc_5
